DFReader.py

```diff
@@ -175,10 +188,8 @@ def find_time_base(self, gps, first_us_stamp):
 
     def type_has_good_TimeMS(self, type):
         '''The TimeMS in some messages is not from *our* clock!'''
-        if type.startswith('ACC'):
```

Comment:
Not sure I agree with this one; it really is a switch-style construct.

If we did go down this path you could just return the comparison rather than have the "if"..